src/ResponsiveImages/RegisteredResponsiveImages.php

```diff
@@ -50,16 +50,19 @@ public function getSrcset(): string
             })
             ->implode(', ');
 
-        if (config('medialibrary.responsive_images.use_tiny_placeholders')) {
+        $mayAddPlaceholderSvg = config('medialibrary.responsive_images.use_tiny_placeholders')
```

Comment:
Rename this to `shouldAddPlaceholderSvg`. That's more in line with similar functions in other packages.